logpuzzle.py

```diff
@@ -40,14 +62,26 @@ def download_images(img_urls, dest_dir):
     with an img tag to show each local image file.
     Creates the directory if necessary.
     """
+
+    path = os.getcwd()
+    os.mkdir(path+"/"+dest_dir)
```

Comment:
Remember: for joining directories with filenames, use `os.path.join` to be platform-agnostic